benchadapt/python/benchadapt/result.py

```diff
@@ -135,7 +135,9 @@ class BenchmarkResult:
     machine_info: Dict[str, Any] = field(default_factory=_machine_info.machine_info)
     cluster_info: Dict[str, Any] = None
     context: Dict[str, Any] = field(default_factory=dict)
-    github: Dict[str, Any] = field(default_factory=_machine_info.github_info)
+    github: Dict[str, str] = field(
```

Comment:
Could you adjust the docstring for this field based on this PR's behavior?

Reply:
Oh, I forgot that indeed. Thanks for the reminder!

Reply:
Yes, I understand now. I was confusing the two objects here:
- the dataclass instance
- the to_publishable_dict() result